tests/common.py

```diff
@@ -897,10 +897,10 @@ def init_recorder_component(hass, add_config=None):
     _LOGGER.info("In-memory recorder successfully started")
 
 
-async def async_init_recorder_component(hass, add_config=None):
+async def async_init_recorder_component(hass, config={}):
```

Comment:
Let's not have mutable default arguments
```suggestion
async def async_init_recorder_component(hass, config=None):
```

Reply:
Ok, I guess then we can leave the name as it was, essentially reverting this line:
```suggestion
async def async_init_recorder_component(hass, add_config=None):
```